config.yml

```diff
@@ -36,3 +36,6 @@ mps-deploys:
   prefix: static
 fission-experiment-monitoring-dashboard:
   gcs_bucket: fission-experiment-monitoring-dashboard
+glean-dictionary:
+  gcs_bucket: glean-dictionary-dev
+  public: true
```

Comment:
Does this option work?

Reply:
It should after @robotblake deploys the latest version of protodash.